oioioi/base/registration_backend.py

```diff
@@ -63,17 +63,23 @@ def register(self, form):
 
 
 urlpatterns = [
-    re_path(r'^register/$', RegistrationView.as_view(), name='registration_register'),
+    re_path(r'^sign-up/$', RegistrationView.as_view(), name='sign-up'),
+    re_path(r'^register/$', RedirectView.as_view(url='/sign-up/', permanent=True), name='registration_register'),
 ]
 
 if not settings.SEND_USER_ACTIVATION_EMAIL:
     urlpatterns += [
         re_path(
-            r'^register/complete/$',
+            r'^sign-up/complete/$',
             TemplateView.as_view(
                 template_name='registration/'
                 'registration_and_activation_complete.html'
             ),
+            name='sign-up_complete',
+        ),
+        re_path(
```

Comment:
I asked for it, but I must have marked the wrong line, sorry. I like the redirection from `/register/` to `/sign-up/`, but this complete link is a not necessary.